packages/tsconfig-reference/copy/pt/options/watchDirectory.md

```diff
@@ -0,0 +1,10 @@
+---
+display: "watchDirectory"
```

Comment:
Estamos traduzindo os display names também. Você pode traduzir eles, por favor?

Reply:
É que o display name aqui no caso é o nome da option que está traduzida, por isso que não mudei. Porque é exatamente como está no config.

Reply:
https://www.typescriptlang.org/pt/tsconfig#allowJs

No site, vai aparecer o display traduzido e logo em seguida a flag

Reply:
Yeah, exactly, my goal here was to give you the chance to include something which someone might use `cmd + f` to find on the site

Reply:
Ohhh Ok! I'll translate those!